zephyr/CMakeLists.txt

```diff
@@ -868,6 +914,15 @@ zephyr_library_sources_ifdef(CONFIG_COMP_MUX
 	${SOF_AUDIO_PATH}/mux/mux_${ipc_suffix}.c
 )
 
+zephyr_library_sources_ifdef(CONFIG_COMP_MFCC
```

Comment:
@marc-hb @lyakh I copied what's been done earlier but we do have in component directory the similar files, also for math libary, could they be used for cleaner SOF zephyr top level CMakeLists.txt. Similarly as it was in XTOS build.

Reply:
Same comment as above